### ISSUE TITLE
Modal JS Keyboard Trap

### ISSUE THREAD

**Issue by HarshaNaidu11** [SAPTARSHI-coder/EaseMotion-css#7475]
## Description

Escape close and focus traps only initialize via location hash, failing on modals opened via .is-active.
